README.md

```diff
@@ -77,3 +77,6 @@ This is a list of the required environment variables:
 #### MSSQL Server
 
 **SA_PASSWORD** - MSSQL Server "SA" user password
+
+## SQL Server Database Schema
+![Database_Schema](Squirrel_DB_Schema.svg)
```

Comment:
Either use Mermaid for it or find a way to update svg file and add to documentation)

Reply:
DB schema updated with Mermaid